Godeps/Godeps.json

```diff
@@ -26,12 +26,12 @@
 		},
 		{
 			"ImportPath": "cloud.google.com/go/compute/metadata",
-			"Comment": "v0.1.0-115-g3b1ae45",
+			"Comment": "v0.1.0-115-g3b1ae453",
```

Comment:
Can we remove these comment changes? (will hopefully be resolved once and for all soon)

Reply:
Done!